Tensile/KernelWriterAssembly.py

```diff
@@ -13217,10 +13543,11 @@ def closemovaccVgpr(self,kernel, backupSgpr):
   ##############################################################################
   def generateOddCodeForStoreCInUnroll(self,kernel):
     kStr = ""
-    kStr += self.comment("Odd code for StoreCInUnroll")
+    # no Odd code for StoreCInUnroll so far
+    #kStr += self.comment("Odd code for StoreCInUnroll")
     # add StoreCIndex0 by inc1
-    inc1 = self.getAccVgprInc1(kernel)
-    kStr += inst("s_addk_i32", sgpr("StoreCIndex0"), inc1,  "")
+    #inc1 = self.getAccVgprInc1(kernel)
+    #kStr += inst("s_addk_i32", sgpr("StoreCIndex0"), inc1,  "")
```

Comment:
Looks like this function can be removed and calls cleaned up

Reply:
I kept it for future use...
I can remove it for now.

Reply:
Deleted.